src/views/report/ScriptReportDetail/ScriptExecutionDetailActions.tsx

```diff
@@ -287,15 +287,16 @@ function ScriptExecutionDetailActions<ColumnNames>({
                                             <TableCell component="th" scope="row" className={classes.thCell}>
                                                 {parameter.name}
                                             </TableCell>
-                                            <TableCell className={classes.thCell}>
+                                            <TableCell>
                                                 <Box
                                                     display="flex"
+                                                    justifyContent="space-between"
                                                     alignItems="center"
                                                 >
                                                     {parameter.rawValue}
                                                     {
                                                         (parameter.name === 'request') ? (
-                                                            <Box marginLeft="90px">
+                                                            <Box display="flex">
```

Comment:
Why adding a display flex in a flex-item ? Just add justifyContent="space-between" in parent Box